spec/lib/tito_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "climate_control"
+require "json"
+require "webmock/rspec"
+require "tito"
+
+describe Tito do
+  around do |example|
+    ClimateControl.modify TITO_AUTH_TOKEN: 'bar' do
+      example.run
+    end
+  end
+
+  describe "#event_list" do
+    it "returns a list of events" do
+      event_data = { "title" => "event info" }
```

Comment:
In one case you use a method stub in which you actually do everything for it. In the other you use a webmock stub so that httparty is still actually hitting a request. This seems like you're testing in two different ways.

I would personally test it through the webmock in both cases but the important thing is taht if you're going to stub and object, do the same in both cases. If you're going to use webmock, do the saem in both cases.